sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/SendOptions.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus.implementation;
+
+
+import com.azure.core.annotation.Fluent;
+
+/**
+ * The set of options that can be specified when sending a set of events to influence the way in which events are sent
+ * to the Event Hubs service.
+ */
+@Fluent
+public class SendOptions {
```

Comment:
The message class itself has "PartitionKey". Partition id is not a concept in SB. Do we need this class?